website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -474,7 +501,7 @@ mutation {
 }
 ```
 
-**Query with a where filter** 
+#### Query with a where filter
 
 The `where` filter takes a list argument (or a string for a single input). Depending on the object you are filtering, there are a couple of parameters:
```

Comment:
Not related to this PR, but I think at some point I think we should actually make a separate page for query parameters in general, since the same parameters apply to both APIs. Then we could put all the details about how to write a where filter, the purpose of a metric alias, and what time grain does in one place, and link to that from each of the API pages where we give examples of the specific syntax for that API. Then we wouldn't have to explain the same things on both API pages. I think that would have helped with the issue yesterday, too, where people couldn't find the docs for where filters.

Reply:
created a github issue for this; https://github.com/dbt-labs/docs.getdbt.com/issues/6781

Reply:
good shout!